Doxyfile

```diff
@@ -38,7 +38,7 @@ PROJECT_NAME           = "Azure SDK for Embedded C"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         =
+PROJECT_NUMBER         = "1.0.0-preview.1"
```

Comment:
@danieljurek - will we have to remember to update this for each release version update? Any way to sync it up with https://github.com/Azure/azure-sdk-for-c/blob/39d9e43c16fd8cea9fafd9c1add053ac21c1d431/sdk/core/core/inc/az_version.h#L21

Reply:
Yes. This is something that the release automation will take care of. It should be read from `az_version.h`